sites/bizbash/server/styles/index.scss

```diff
@@ -475,6 +475,16 @@ body {
       }
     }
   }
+
+  &--featured-venues-suppliers {
+    & .item-list {
```

Comment:
This doesn't follow the BEM standard. Change to `#{ $self } {` instead of `& .item-list {` (see `&--directory-pinned-items` for example)